SPECS/python-absl-py/python-absl-py.rpmlintrc

```diff
@@ -0,0 +1,2 @@
+# These are not real spelling errors...
```

Comment:
We don't use rpmlint- we can safely delete this file.

Reply:
deleted.